roles/workstation/files/cv.sudoers

```diff
@@ -0,0 +1 @@
+%Local\ Admins ALL=(ALL) ALL
```

Comment:
I haven't looked at @the-maldridge's expansion of the groups, but either way I believe that this should be variabalized and put into netlogin? That's my preference, but I'm open to keeping things the way they are/changing this at a later time...